scripts/lint_naming_conventions.py

```diff
@@ -89,6 +93,22 @@ def parse_file_name(in_dir, in_file):
     return 0
 
 
+# Ensure all jobs have a retention policy
+def check_retention(job, in_file):
```

Comment:
This function doesn't fail on `build-discarder: {}`, is that intentional? Based on [1] I'd have thought this would want to require either `days-to-keep` or `num-to-keep` is set. Is it worth putting limits on what these values can be?

[1] https://docs.openstack.org/infra/jenkins-job-builder/properties.html#properties.build-discarder

Reply:
Yep, I think we should check specific values ... with at minimum value > 0 being acceptable.